io_export_objex2/blender_version_compatibility.py

```diff
@@ -18,7 +18,7 @@ def make_annotations(clazz):
     if bpy.app.version < (2, 80):
         return clazz
     for cls in clazz.__mro__:
-        bl_props = {k: v for k, v in cls.__dict__.items() if isinstance(v, tuple) and len(v) == 2 and inspect.isbuiltin(v[0]) and isinstance(v[1], dict)}
+        bl_props = {k: v for k, v in cls.__dict__.items() if (isinstance(v, tuple) and len(v) == 2 and inspect.isbuiltin(v[0]) and isinstance(v[1], dict)) or isinstance(v, bpy.props._PropertyDeferred)}
```

Comment:
I think it would be better to attach each condition to the right blender version, as accessing `bpy.props._PropertyDeferred` before 2.9x would probably raise an error

Kinda like this

```suggestion
        bl_props = {k: v for k, v in cls.__dict__.items() if (bpy.app.version < (2, 90) and isinstance(v, tuple) and len(v) == 2 and inspect.isbuiltin(v[0]) and isinstance(v[1], dict)) or (bpy.app.version >= (2, 90) and isinstance(v, bpy.props._PropertyDeferred))}
```

But this line is becoming unreadable... Maybe something like this:
```py
if bpy.app.version < (2, 90):
        isPropertyDeferred = lambda v: (isinstance(v, tuple) and len(v) == 2 and inspect.isbuiltin(v[0]) and isinstance(v[1], dict))
else:
        isPropertyDeferred = lambda v: isinstance(v, bpy.props._PropertyDeferred)
for cls in clazz.__mro__:
        bl_props = {k: v for k, v in cls.__dict__.items() if isPropertyDeferred(v)}
```

Reply:
Opted to apply something more akin to the more readable suggestion, targeting version 2.93 specifically since up until 2.92 the Tuple type detection worked fine.